rules/autoconsent/whitepages.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "whitepages.com",
+    "cosmetic": true,
+    "prehideSelectors": [".cookie-wrapper, .cookie-overlay"],
+    "detectCmp": [{ "exists": ".cookie-wrapper" }],
```

Comment:
this selector sounds quite generic, so I would suggest limiting this rule to one site only using `urlPattern`

Reply:
See here the addition of urlPattern